packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.types.ts

```diff
@@ -49,11 +50,6 @@ export interface ICommandBarProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   overflowButtonAs?: IComponentAs<IButtonProps>;
 
-  /**
-   * Menu props to be passed to overflow elipsis
-   */
-  overflowMenuProps?: Partial<IContextualMenuProps>;

```

Comment:
Isn't this a breaking change?

Reply:
this prop never actually worked :D it was left in when moving from experiments to 6.0